### DISCUSSION TITLE
Connecting to Trino

### DISCUSSION THREAD

**Discussion by maayaniti** [great-expectations/great_expectations#5723] [Support]
Hi, I followed this [guide](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/database/trino/) to connect to Trino and tried testing it using a connection_string with a specific catalog and schema, that has only one table in it.
When trying to test the connection with `context.test_yaml_config(yaml.dump(datasource_config))` I see many queries running on trino -  
```
PREPARE st_1234someguid FROM SELECT "table_name"
FROM "information_schema"."views"
WHERE "table_schema" = ?
```
and then `EXECUTE st_1234someguid USING 'some_user'`

It looks like it's ignoring the specific schema I specified and trying to collect data from other schemas.
How can I set the datasource_config to query only from one schema?

**Comment by pcontrerasnun** [comment]
It works well for me using `trino://<USERNAME>:<PASSWORD>@<HOST>:<PORT>/<CATALOG>/<SCHEMA>`
Maybe check your GE version?
